src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -40,8 +40,9 @@ string GetVisualStudio2017Directory ()
 		public string XABuildExe {
 			get {
 				if (IsUnix) {
-					if (!string.IsNullOrEmpty (Environment.GetEnvironmentVariable ("USE_MSBUILD"))) {
-						RunningMSBuild = true;
+					RunningMSBuild = true;
+					if (!string.IsNullOrEmpty (Environment.GetEnvironmentVariable ("USE_XBUILD"))) {
```

Comment:
Shouldn't we stick with `$USE_MSBUILD` for now, and when `$USE_MSBUILD` is unset or `0` we assume `xbuild`

*Eventually* we'll make `msbuild` the default everywhere, and *then* it might -- *might* -- make sense to use `$USE_XBUILD` instead of `$USE_MSBUILD`...but *now*?